src/aspire/nfft/__init__.py

```diff
@@ -39,34 +44,49 @@ def _try_backend(backend):
         We only check for working imports here to keep things simple and lightweight.
         If imports are working but the actual backend is not then we have bigger problems anyway.
         """
+
         logger.info(f"Trying NFFT backend {backend}")
         plan_class = None
-        if backend == "finufft":
+        msg = None
+        if backend == "cufinufft":
+            try:
+                # Note we expect the cu library is in LD path.
+                #   That is managed by pip if using a wheel,
+                #   or the user if they have built from source.
+                from aspire.nfft.cufinufft import cuFINufftPlan
+                plan_class = cuFINufftPlan
+            except Exception as e:
+                msg = str(e)
+                pass
+
+        elif backend == "finufft":
             try:
                 from finufftpy import nufft1d1
                 from aspire.nfft.finufft import FINufftPlan
                 plan_class = FINufftPlan
-            except ImportError:
+            except Exception as e:
+                msg = str(e)
                 pass
 
         elif backend == "pynfft":
             try:
                 from pynfft.nfft import NFFT
                 from aspire.nfft.pynfft import PyNfftPlan
                 plan_class = PyNfftPlan
-            except ImportError:
+            except Exception as e:
+                msg = str(e)
                 pass
 
         if plan_class is None:
-            logger.info(f"NFFT backend {backend} not usable")
+            logger.info(f"NFFT backend {backend} not usable:\n\t{msg}")
         else:
-            logger.info(f"NFFT backend {backend} usable")
+            logger.info(f"NFFT backend {backend} usable.")
             return plan_class
 
     backends = OrderedDict((k, _try_backend(k)) for k in config.nfft.backends)
     try:
         default_backend = next(k for k, v in backends.items() if v is not None)
-        logger.error(f'Selected NFFT backend = {default_backend}')
+        logger.error(f'Selected NFFT backend = {default_backend}.')
```

Comment:
`logger.error` should be `logger.info`?

Reply:
Reading the code, do you think we should reach that line in normal operation?

Perhaps it needs a better message.

Reply:
Regardless, changed.